api/v1alpha1/gatekeeper_types.go

```diff
@@ -163,6 +163,13 @@ type AuditConfig struct {
 	// +kubebuilder:default=Disabled
 	//nolint:lll
 	AuditEventsInvolvedNamespace Mode `json:"auditEventsInvolvedNamespace,omitempty"`
+
+	// PodAnnotations is a map of additional annotations to be added to the audit deployment pods.
+	// When the same keys are defined in the global podAnnotations, these annotations will take precedence.
```

Comment:
I think providing the path might be clearer?

```suggestion
	// When the same keys are defined here and in spec.podAnnotations, these annotations take precedence.
```